### PR TITLE
Query the baseUrl when calling createOrUpdate.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#1292

### CODE REVIEW

File: packages/client-node/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "type": "module",
   "scripts": {
     "build": "yarn lint && tsc -p tsconfig.json",
-    "integration-test": "docker compose up -d && ./scripts/integration_test_db_setup.sh && NODE_OPTIONS=--loader=ts-node/esm jasmine --config=jasmine-integration.json ; docker compose down",
+    "integration-test": "docker compose up -d && ./scripts/integration_test_db_setup.sh && NODE_OPTIONS=--loader=ts-node/esm jasmine --config=jasmine-integration.json",
```
Comment (gdethier) [inline packages/client-node/package.json:10]: You probably forgot to revert this change.

**Review by gdethier — APPROVED** [review]
Good job! I left one remark.

File: packages/client-node/package.json (hunk 1 above)
Comment (gdethier) [reply packages/client-node/package.json]: Done

Comment (benoitdevos) [reply packages/client-node/package.json]: Indeed, thanks.
